pwa/src/pages/configurations/index.tsx

```diff
@@ -9,7 +9,7 @@ const IndexPage = () => {
         <div className="row">
           <div className="col-12">
             <div className="page-top-item">
-              <ConfigurationsTable/>
+              <Configurations/>
```

Comment:
Hier mag de logica en button komen, in plaats van dat er een `<Configurations/>` component wordt ingeladen.

Tip: wat nog mooier zou zijn is om een component te maken genaamd `ConfigurationExportButton` en daar de button in te maken en dat component hier in te laden. Note: dat component hoeft dus niet een hele tabel te hebben etc.

Bel me maar even